site/docs/v1/tech/migration-guide/_what-next.adoc

```diff
@@ -1,22 +1,22 @@
-You now have your users migrated. Congratulations! What is next?
+You now have your users migrated, or a plan to do so. Congratulations! What is next?
 
 You need to migrate the other configuration, as mentioned in <<Other Entities>>. Since the type of configuration varies, it is hard to provide a full list of how to import these items, but the general pattern will be:
 
-* Identify the corresponding FusionAuth functionality.
-* Configure it in your FusionAuth instance, either manually or script it using the client libraries or API.
-* Update your application configuration to use the FusionAuth functionality.
+* Identify corresponding FusionAuth functionality.
+* Configure it in your FusionAuth instance, either manually or by scripting it using the client libraries or API.
```

Comment:
Should we link the client libraries or API here

Reply:
Good idea.